ratis-server/src/main/java/org/apache/ratis/server/raftlog/segmented/SegmentedRaftLogCache.java

```diff
@@ -383,6 +383,7 @@ TruncationSegments purge(long index) {
         list.forEach(LogSegment::evictCache);
         List<SegmentFileInfo> toDelete = list.stream().map(SegmentFileInfo::newClosedSegmentFileInfo)
             .collect(Collectors.toList());
+        LOG.info("final toDelete: {}", toDelete);
```

Comment:
Print `LogSegment` instead of `SegmentFileInfo`.  It is easier to read.
```java
        LOG.info("{}: toDelete segment: {}", name, list);
```

Reply:
I have reverted this line